collector/infiniband_linux_test.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 The Prometheus Authors
```

Comment:
Might be worth considering a test like what exists in the ZFS code: https://github.com/prometheus/node_exporter/pull/432/files#diff-1a2aec762c5ab3c1342874a30dd41671R216 or even something like what the bonding test does: https://github.com/prometheus/node_exporter/blob/master/collector/bonding_linux_test.go

Basically, checking that you're reading the fixtures correctly.